packages/flutter_test/lib/src/window.dart

```diff
@@ -518,6 +519,20 @@ class TestWindow implements ui.SingletonFlutterWindow {
   }
 }
 
+/// Convenient methods for auto reset test values
+// Use an `extension`, such that classes implementing TestWindow will not be affected
```

Comment:
`//` should be `///`.  Also there should be an empty line of `///` above this line.

Reply:
"Uses an `extension` so that classes implementing TestWindow will not be affected."